Automattic/wp-calypso
# Map Block: A8C Mapbox Account

#### Changes proposed in this Pull Request

The original Map block used styles associated with my personal Mapbox.com account. This PR switches to A8C's account. 

#### Testing instructions

Test locally or with JN, and verify that map styles continue to work.

JN link: https://jurassic.ninja/create/?gutenberg&gutenpack&shortlived&jetpack-beta&calypsobranch=update/map-a8c-mapbox-account&branch=master
